test/unittests/parser_test.cpp

```diff
@@ -285,6 +285,42 @@ TEST(parser, global_multi_const_inited)
     EXPECT_EQ(module.globalsec[1].init.value, uint32_t(-1));
 }
 
+TEST(parser, export_single_function)
+{
+    const auto section_contents = bytes{0x01, 0x03, 'a', 'b', 'c', 0x00, 0x42};
+    const auto bin =
+        bytes{wasm_prefix} + uint8_t{0x07} + uint8_t(section_contents.size()) + section_contents;
+
+    const auto module = parse(bin);
+    ASSERT_EQ(module.exportsec.size(), 1);
+    EXPECT_EQ(module.exportsec[0].name, "abc");
+    EXPECT_EQ(module.exportsec[0].type, ExportType::Function);
+    EXPECT_EQ(module.exportsec[0].index, 0x42);
+}
+
+TEST(parser, export_multiple)
```

Comment:
Can you add a test where the section contains `(export "memory" (memory 1))`  and trying to find "memory" as a function?

Reply:
added checks to api.find_exported_function test, is that ok?

Reply:
Yes, I just made the comment at the wrong place.